package.json

```diff
@@ -27,8 +27,8 @@
   "scripts": {
     "watch": "babel --plugins transform-es2015-modules-umd src --watch --ignore tests --out-dir ./dist",
     "prepublish": "babel --plugins transform-es2015-modules-umd src --ignore tests --out-dir ./dist",
-    "lint": "eslint ./src",
-    "lintfix": "eslint ./src --fix",
+    "eslint": "eslint 'src/**/*.js'",
+    "prettier": "prettier --write 'src/**/*.{js,jsx,json,css}'",
     "testonly": "mocha $npm_package_options_mocha",
     "test": "npm run lint && npm run testonly",
```

Comment:
@lcustodio you just need to update this script (from `lint` to `eslint`)

Reply:
`lint` it is 😉

Reply:
But you need to change in the script: `"test": "npm run lint && npm run testonly"`, you know?

Reply:
Sorry, I wasn't clear. I reverted the script `eslint` back to `lint`, so the former `test` works. 

```json
"scripts": {
    "----> lint <----": "eslint ./src",
    "prettier": "prettier --write 'src/**/*.{js,jsx,json,css}'",
    "testonly": "mocha $npm_package_options_mocha",
    "test": "npm run ----> lint <---- && npm run testonly",
  }
```

Reply:
Ahhh ok, I did't see your last commit 👍